### PR TITLE
[3.2.0] Update setting up MySQL document

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md [hunk 1]
```diff
@@ -200,6 +200,9 @@ Follow the  instructions below to change the type of the default datasources.
         ``` java
         driver="com.mysql.cj.jdbc.Driver"
         ```
+    
+    !!! note "Recommendation"
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md:204]: @vithu30 , this note needs to go into the 3.1.0 docs as well.

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@vithu30 , please send these changes to the master branch as well.

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md (hunk 1 above)
Comment (vithu30) [reply en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md:204]: Added to 3.1.0 with PR - https://github.com/wso2/docs-apim/pull/2074

**Comment by vithu30** [comment]
@Mariangela  Fixed in master with https://github.com/wso2/docs-apim/pull/2102

**Pull request merged**